lib/index.html

```diff
@@ -99,16 +99,27 @@
             <div class="box text-center">
               <h3><hero-tick type="success" class="space-right-tiny"></hero-tick> Burn Complete!</h3>
               <p class="soft">Safely ejected and ready for use</p>
-
-              <div class="space-vertical-large">
-                <hero-button ng-click="app.restart({ preserveImage: true })">
-                  <span class="glyphicon glyphicon-repeat" aria-hidden="true"></span>
-                  Burn image again
-                </hero-button>
-                <hero-button ng-click="app.restart()">
-                  <span class="glyphicon glyphicon-cd" aria-hidden="true"></span>
-                  Burn another image
-                </hero-button>
+              <div class="col-xs center-xs space-vertical-large">
```

Comment:
Modifiers such as `center-xs` are only meant to be added to `row` elements. This pattern happens in various places below.